src/azure-cli/azure/cli/command_modules/synapse/tests/latest/assets/kqlScript.kql

```diff
@@ -0,0 +1 @@
+Logs | where TimeStamp > ago(1h) | take 10
```

Comment:
If you want to add `.kql` file for test, I'm afraid you have to update our CI build script:
https://github.com/Azure/azure-cli/blob/fc2822a6e4fb76c6a9fca23267b0faddd56bcab8/scripts/ci/build.sh#L135-L168

Reply:
Nice catch.